### PR TITLE
Creating ArgoCD Project and app for Fybrik

### PR DESCRIPTION
**Pull request by eletonia**
This PR replaces the previous `mesh-for-data` ArgoCD project and app with the `fybrik` ArgoCD project and app
These are to be deployed on the the MOC Smaug cluster

@4n4nd @ronenkat

### CODE REVIEW

**Review by 4n4nd — CHANGES_REQUESTED** [review]
Can you move `envs/moc/zero/fybrik/kustomization.yaml` to `envs/moc/smaug/fybrik/`?

File: projects/fybrik.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: fybrik
+  labels:
+    project-template: global
+spec:
+  destinations:
+    - namespace: 'fybrik-*'
+      server: 'https://api.zero.massopen.cloud:6443'
```
[the remark this answers is not in the document]
Comment (eletonia) [reply projects/fybrik.yaml]: @4n4nd Thanks, I changed the server url to the smaug one

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**